src/python/grpcio_tests/tests_aio/unit/sync_server.py

```diff
@@ -26,9 +26,15 @@
 # Change for an asynchronous server version once it's implemented.
 class TestServiceServicer(test_pb2_grpc.TestServiceServicer):
 
+    CALL_DELAY = 0.2
+
     def UnaryCall(self, request, context):
         return messages_pb2.SimpleResponse()
 
+    def EmptyCall(self, request, context):
```

Comment:
I think this discussion was somewhere else already but I don't find it. Why can't we call this `SleepCall` or `DelayCall` or something like that?

Reply:
This server's implementation depends on the definition of the protobuf, so the methods must be called as per that definition. Calling it something else (``TimeoutCall``, ``DelayedCall``, etc.) requires us to come up with a new protobuf, and a new server object that implements it. So in order to reuse the sever we already have, I implemented another of the mehtods that was left free.